templates/contrib/jdbc/README.md

```diff
@@ -14,46 +14,24 @@ It adds a extra layer on top of the base image which contains:
   - needs to respect the JBoss EAP module structure: i.e:
             */extensions/modules/org/postgresql96/main/*
 - the install.properties which contains the JBoss module information, this file is populated during the build image
- , [here](modules/kie-custom-jdbc-driver/added/base_install.properties) you can find the base file used.
+ , [here](cekit/modules/kie-custom-jdbc-driver/added/base_install.properties) you can find the base file used.
 
 - the install.sh script which is responsible to configure the JBoss module: [install.sh](modules/kie-custom-jdbc-driver/added/install.sh)
 
-## Before you begin
-
-### Pre-requisites
-
-To interact with this repo you should install the CEKit 3.2.0:
-
-#### Installing and Configure CeKit
-In this section you'll , step by step, how to install CeKit on Fedora, for other systems please refer to the documentation: 
-To be possible to install the CeKit using dnf, we need to enable it's dnf repository:
-
-Install [CeKit](https://docs.cekit.io/en/latest/handbook/installation/instructions.html#other-systems) 3.2 using virtualenv: 
-
-To install, see the following commands:
-
-```bash
-$ mkdir ~/cekit-3.2
-$ vritualenv ~/cekit-3.2
-$ source ~/cekit-3.2/bin/activate # tip create an alias for it, e.g. activate-cekit-3.2
-$ pip install cekit==3.2.0 
-$ pip install odcs docker docker-squash behave
-```
-
 
 The drivers images are based on the supported configurations for Red Hat Process Automation Manager, which are:
 
 | Database    | DB Version              | JDBC Driver jar/version         |
 |-------------|-------------------------|---------------------------------|
 | IBM DB2     | 11.1                    | jcc-11.1.4.4.jar                |
-| MariaDB     | 10.2                    | mariadb-java-client-2.3.0.jar   |
-| MS SQL      | 2014,2016               | mssql-jdbc-7.2.2.jre11.jar       |
+| MariaDB     | 10.2                    | mariadb-java-client-2.4.0.jar   |
+| MS SQL      | 2014,2016               | mssql-jdbc-7.2.2.jre11.jar      |
 | Oracle DB   | 12c RAC                 | example: ojdbc7.jar             |
 | MySQL       | 5.7                     | mysql-connector-java-8.0.12.jar |
 | PosgtgreSQL | 10.1, 10.1 Enterprise   | postgresql-42.2.5.jar           |
 | Sybase      | 16.0                    | jconn4-16.0_PL05.jar            |
 
-For more information, please visit the RHPAM [compatibility matrix](https://access.redhat.com/articles/3405381#RHPAM74).
+For more information, please visit the RHPAM [compatibility matrix](https://access.redhat.com/articles/3405381#RHPAM77).
```

Comment:
This change will conflict with the PR #445 . Maybe we should update the table here and reject 445?

Reply:
can be :)

Reply:
merged #445